protocol/error.go

```diff
@@ -15,13 +15,33 @@ const (
 	ErrorMalformedClientMessage
 )
 
+const (
+	Passed ErrorCode = iota + 20
```

Comment:
To clarify, in coniks-java I chose to use `10`s error codes to denote server-side errors, while `20`s error codes denote client-side errors.